pact/Pact/Core/IR/Eval/Direct/Evaluator.hs

```diff
@@ -1763,24 +1763,15 @@ rawSqrt info b _env = \case
     return (VLiteral (LDecimal (f2Dec result)))
   args -> argsError info b args
 
--- Todo: fix all show instances
+showPactValue :: MonadEval b i m => i -> PactValue -> m T.Text
+showPactValue info pv = do
+  sz <- sizeOf SizeOfV0 pv
+  chargeGasArgs info $ GConcat $ TextConcat $ GasTextLength $ fromIntegral sz
+  pure $ Pretty.renderCompactText pv
```

Comment:
Small bikesheddey nitpick: if i'm calling `showPactValue`, i don't assume any rendering or compaction: I assume it's using the show instance on pact values. Maybe we keep `renderPactValue` as a name, because it implies there's work being done on the value to produce text.

Reply:
Good point, I agree the current name is slightly misleading.